frontend/assets/stylesheets/components/_bundle-thank-you.scss

```diff
@@ -62,7 +62,12 @@
     }
 
     input[type=email] {
```

Comment:
Aha, I mean, `type="email"` with quotation marks, instead of without (line 30 of [this example](https://developer.mozilla.org/en/docs/Web/CSS/Attribute_selectors#Example)). Sorry, that was really not very clear, my bad 🙃.